pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiQuestionCached.java

```diff
@@ -311,6 +314,11 @@ public Map<Long, List<Long>> collectOrderedCompositeChildIdsByParentIdsAndTimest
         }
     }
 
+    @Override
+    public boolean deleteBaseQuestion(long questionId) {
+        throw new DDPException("Not implemented for cached version");
+    }
+
```

Comment:
If the delete methods are intended to be an optional operation, I'd support this. If we go down this route, I'd suggest
* Updating the javadoc to clearly state the delete methods are optional
* Use a `UnsupportedOperationException` instead of a `DDPException`

If the delete methods are not intended to be optional, and just not implemented, using a `default` implementation is a anti-pattern we should avoid.